### PR TITLE
Responsive Design and CSS Frameworks: Using Bootstrap

### PR DESCRIPTION
**Pull request by nikoescobal**
## Newsweek Clone: Responsive Design and CSS Frameworks - Part 2 (Using Bootstrap)

In this project:
- An exact replica of [Newsweek](newsweek.com) built with a mobile-first approach.
- Bootstrap's CSS classes and grid system are utilized to make the website fully responsive across all screen sizes.
- CSS flex, grid, positioning, media queries, and fluid/responsive web design principles are also applied. 
- This document validates as CSS level 3 + SVG by W3C CSS Validator and is validated by Jigsaw for HTML.

I initially struggled with this, esp. since I did this solo, but I did my best and am proud of my work. Hope you like it.

Thank you
~Nik :heart:

### CODE REVIEW

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Comment by umairarshadbutt** [comment]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @nikoescobal
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ]  `U.S.`,  `World Business`, `Tech & Science`, `culture`, and `Sports` sections are missing. Please add these section in the page.
- [ ] Navbar is not fixed in the clone as compared to the original page.

### Manage the spaces
|[Source Page](https://thenextweb.com/)|[Clone Page](https://rawcdn.githack.com/nikoescobal/newsweek-bootstrap-clone/16ebf8d2043029aac9959f2f7fcae2b911bb07ed/index.html)|
|:---:|:---:|
|space-original|space-clone|
- [ ] Manage the content according to the original page.

### Update the current Date

|[Source Page](https://thenextweb.com/)|[Clone Page](https://rawcdn.githack.com/nikoescobal/newsweek-bootstrap-clone/16ebf8d2043029aac9959f2f7fcae2b911bb07ed/index.html)|
|:---:|:---:|
|date-orginal|date-clone|

- [ ] Update current date.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by nikoescobal** [comment]
@umairarshadbutt Thank you for reviewing my code, and for listening to my case. As we agreed, I will be resubmitting my project as is.

File: index.html [hunk 1]
```diff
@@ -1,10 +1,2413 @@
...
+
+<body class="mx-auto">
+  <header class="position-relative">
+    <nav class="top-nav navbar bg-red py-3 py-md-3 py-sm-3 py-lg-4 pt-lg-4 pb-lg-3 justify-content-between">
+      <div class="d-none d-lg-block">
+        <div class="text-white mb-2">
+          <img src="https://g.newsweek.com/img/weather/34.png" alt="Fair / Windy" class="weather-icon" />
+          31° Makati, PH <i class="fas fa-chevron-right"></i>
+        </div>
+        <span class="text-white d-none d-lg-block ml-1">Mon, Aug 31, 2020</span>
```
Comment (arslanbisharat) [inline index.html]: - [ ] As requested from previous TSE, please update date

File: index.html [hunk 2]
```diff
@@ -1,10 +1,2413 @@
...
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```
Comment (arslanbisharat) [inline index.html:3011]: - [ ] There is a break-line on the original page that is missing in your project. 
Please have a look. 
Your Work
original work

File: index.html [hunk 3]
```diff
@@ -1,10 +1,2413 @@
...
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-lg-5">
+              <div class="newsletter position-relative mt-2 pt-3 pt-md-1 pt-lg-1 pb-lg-0 m-lg-0 border-bottom">
+
+                <h6 class="text-red font-weight-bold text-uppercase pb-1 mt-2 pt-3 pt-md-1 pt-md-2 p-lg-0">
+                  newsletter
```
Comment (arslanbisharat) [inline index.html:2985]: - [ ] Photo is missing under newsletter as present in original page
chrome-capture (1)

File: index.html [hunk 4]
```diff
@@ -1,10 +1,2413 @@
...
+                </li>
+                <li class="m-1 px-1">
+                  <a href="#"></a>Archive
+                </li>
+                <li class="m-1 px-1">
+                  <a href="#"></a>Announcements
+                </li>
+                <li class="m-1 px-1">
+                  <a href="#"></a>Consent preferences
+                </li>
```
Comment (arslanbisharat) [inline index.html]: - [ ] Add l after each link. 
chrome-capture (2)

File: index.html [hunk 5]
```diff
@@ -1,10 +1,2413 @@
...
+                        people who lost their jobs from the coronavirus
+                        pandemic are still out of work.
+                      </p>
+                    </div>
+                  </figcaption>
+                </figure>
+              </article>
+            </div>
+
+            <div class="my-turn order-4 position-relative d-md-none pt-4">
```
Comment (arslanbisharat) [inline index.html:531]: - [ ] Have a look at the original project. Make it static

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes
I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.:worried:
It's okay. We are here to learn. Don't get discouraged. Work more hard.
 **Please have a look at the suggestions to get more understanding. :eyes:**

- [ ] I couldn't understand why you didn't change the date. :confused: Change to date from **Mon, Aug 31, 2020,**, to the **current date**

- [ ] Make the center static as in the original page. 

- [ ] Make your footer exactly the same as present on the original page. 

- [ ] Add image under newsletter and increase left width.

- [ ]  Please correct these
chrome-capture (3)

**Please make the changes and request re-review for the given project**
Stay safe:innocent:
### Happy Coding
-------------------------------------------------------------------------------------------------------------------------------------------

**Regards,
Arslan Bisharat
****slack: arslanbisharat******

**Comment by nikoescobal** [comment]
I've implemented relevant changes. @umairarshadbutt  @arslanbisharat I'm sorry, but this request to have the date changed is not objectively useful or relevant. I don't see how this helps improve my code. 

I asked my standup teammates and other students and they all told me this was not an issue that prevented their projects from being approved. In fact, many projects had placeholders, like Lorem Ipsum for the content, including the date section.

As TSE's, you have the potential to really help students improve. I sincerely ask that you give valid and objectively useful feedback based on best-practices, and not subjective change requests that are based on your preferences. 

Thank you.

**Review by javitocor — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED ♻️ 

Good job so far with this project 👍  but still some changes left to get approved. It is almost done, just a little bit more. ⏩ 
After all the requirements are made, re-submit the project for a new code review.

If you have any question or doubt do not hesitate to contact me on Slack.

Check my comments below 👇 

- There are still some sections left to build: US, World, Business, Tech&Science, Culture and Sports. Please add all of them 💪 
- The navigation bar should be fixed or sticky. Make it happen.
- Add the | after every li in the footer
- Change the date, it is a previous requirement.

Good luck 😃

**Comment by nikoescobal** [comment]
@javitocor https://rawcdn.githack.com/nikoescobal/newsweek-bootstrap-clone/29384df0ca08c064180a6aa2a533e869dc4b06f7/index.html I'm not sure if you're looking at the correct project, because all those things are there. The navbar is fixed, there is a border after every li in the footer, and the date has been updated as well. As for the sections you mentioned, as I previously mentioned to the other TSE, the Newsweek website is constantly updating. The version I modeled my code after did not contain those sections. I don't see any reason to build them out, especially if I've already accomplished the purpose of this project, which is to utilize Bootstrap extensively.

**Comment by arslanbisharat** [comment]
Hello! We are really sorry you didn't catch what we were trying to say 

- [ ] There are still some sections left to build: US, World, Business, Tech&Science, Culture, and Sports. Please add all of them 
chrome-capture (20)
chrome-capture (19)
chrome-capture (18)
chrome-capture (17)

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

**It is crystal clear that you've been so careful in implementing this project. I really love how much attention you've paid to the details. However, there's one major feature that's stopping me from approving your project. Please observe carefully and submit another review request form when you're done.**

As mentioned by the previous TSE, there's a major section missing out on your project. I will try to point it out with a detailed illustration. The section is located after Editor's Picks (_see image below_)

**Here's your copy:**
Screenshot_2020-09-16 Newsweek Clone


**Here's source copy:**
Screenshot_2020-09-16 Newsweek


**I hope you now see what the previous TSE meant the issue he raised. Kindly implement the feature and resubmit for re-review.**

With respect, 
Taofeek Olalere 
GitHub: @teekaytech
Slack: @taofeekolalere

**Comment by nikoescobal** [comment]
@teekaytech Thank you for the kind words. Before anything else, I'd really appreciate it if you could make a consideration for this case. As you can see in the image below, this is the version I was modeling my project after (it's also in the README)..

The Newsweek site updates quite frequently, and so these sections come and go. It is with this fact in mind that I opted not to build those sections.

The case I am trying to make is that I feel that the work I've put out is a reflection of my understanding and ability to utilize the Bootstrap framework, which is the true objective of the project. 

If you still strongly feel that I need to build out those sections, then I will do so. Hoping for your kind consideration on the matter. 

Newsweek-screenshot

**Comment by teekaytech** [comment]
> @teekaytech Thank you for the kind words. Before anything else, I'd really appreciate it if you could make a consideration for this case. As you can see in the image below, this is the version I was modeling my project after (it's also in the README)..
> 
> The Newsweek site updates quite frequently, and so these sections come and go. It is with this fact in mind that I opted not to build those sections.
> 
> The case I am trying to make is that I feel that the work I've put out is a reflection of my understanding and ability to utilize the Bootstrap framework, which is the true objective of the project.
> 
> If you still strongly feel that I need to build out those sections, then I will do so. Hoping for your kind consideration on the matter.
> 
> Newsweek-screenshot


Hello @nikoescobal
I totally understand your point and based on what I've seen in the project, it is clear that you understand the concepts of responsive design. I want you to know that TSEs are also students like you and as a result of that, we have a lot of things in common. Most importantly, we all follow instructions. 

For every project, there's a standard set of requirements that must be satisfied. These requirements are microverse standard which cannot be bent by any student (TSEs inclusive). For this particular project, that every section is one of the requirements that any TSE must confirm it's availability.

In summary, you will have to build the section for your project to be complete. Please take some time to implement that section and request another review. I hope you understand my explanations.

Best regards
Taofeek Olalere (@teekaytech )
**Happy coding**

**Comment by nikoescobal** [comment]
I've added all the changes requested.

**Comment by arslanbisharat** [comment]
### Congratulations APPROVED :clap:
Hello I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
Now you can merge the branches and move forward with your curriculum. 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 
**Happy Coding**:smiley:
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) @slack: arslanbisharat
